test/models.py

```diff
@@ -1,4 +1,4 @@
-from psqlgraph import Edge, Node, pg_property
+import psqlgraph
```

Comment:
👍  Thank you for cleaning up the imports throughout this PR to improve readability